docs/cookbook/languages/rust.md

```diff
@@ -269,5 +269,5 @@ runtime-packages = ["libiconv", "gcc"]
 [custom-toolchains]: https://github.com/zmitchell/rust-toolchains
 [esp32]: https://www.espressif.com/en/products/socs/esp32
 [risc-v]: https://en.wikipedia.org/wiki/RISC-V
-[build-concept]: ../../concepts/manifest-builds.md
+[build-concept]: ../../concepts/builds.md
```

Comment:
(unrelated line) note: the `fix: merge main into preview (#284)` commit is not actually a merge commit so I think that's why you got conflicts merging back into main